### PR TITLE
Graph meeting notifications Sample fixes

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/app-installation-using-qr-code/nodejs/.vscode/tasks.json [hunk 1]
```diff
@@ -28,8 +28,7 @@
                     "portOccupancy" // Validate available ports to ensure those debug ones are not occupied.
                 ],
                 "portOccupancy": [
-                    3978, // app service port
-                    9239 // app inspector port for Node.js debugger
```
Comment (Prithvi-MSFT) [inline samples/app-installation-using-qr-code/nodejs/.vscode/tasks.json:32]: second port not required ?

Comment (Mohammed-MSFT) [reply samples/app-installation-using-qr-code/nodejs/.vscode/tasks.json:32]: Yes only server is running 3978, there is no client in this sample

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Mohammed-MSFT** [comment]
**Build PipeLine Executed Successfully**
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8339586605
image

File: samples/graph-meeting-notification/nodejs/helper/graph-helper.js [hunk 2]
```diff
@@ -33,9 +33,11 @@ class GraphHelper {
         let resource = "";
         try {
         applicationToken = await auth.getAccessToken();
+        console.log("application Token--->",applicationToken);
```
Comment (Prithvi-MSFT) [inline samples/graph-meeting-notification/nodejs/helper/graph-helper.js:36]: remove console logs

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipeline
image

**Review by Harikrishnan-MSFT — APPROVED** [review]
Approving

**Pull request merged**